servers/server.js

```diff
@@ -24,87 +24,134 @@ db();
 
 mongoose.set('useFindAndModify', false);
 //Account
-app.get('/add-account', (req, res) => {
-  accountHandler.addNewAccount(res);
+app.post('/add-account', (req, res) => {
+  accountHandler.addNewAccount(req.body).then((msg) => res.json(msg));
 });
 
 app.get('/accounts', (req, res) => {
-  accountHandler.viewAllAccount(res);
+  accountHandler
+    .viewAllAccount()
+    .then((accounts) => res.json(accounts))
+    .catch((err) => res.json(err));
 });
 
-app.get('/accounts/:id', (req, res) => {
-  accountHandler.viewOneAccountById(req, res);
+app.get('/account/:id', (req, res) => {
+  accountHandler
+    .viewOneAccountById(req.params.id)
+    .then((accounts) => res.json(accounts))
+    .catch((err) => res.json(err));
 });
 
-app.get('/accounts/:id/edit', (req, res) => {
-  accountHandler.editAccountById(req, res);
+app.put('/accounts/:id', (req, res) => {
```

Comment:
can we make the view/update/delete single object all singular? like `/account/:id` instead of `/accounts/:id`